.github/workflows/shaptools-ci.yml

```diff
@@ -0,0 +1,103 @@
+name: Formula CI
+# - this workflow will
+# - test python version matrix [2.7,3.6, 3.7, 3.8, 3.9] for coverage 
+# - uploads coverage data to codeClimate --> Results are linked  as a  badges in the README
+# - deliver the package content to the configured repository
+# - submit the new package content to the upstream repository
+on: [push, pull_request]
+env:
+  PACKAGE_NAME: python-shaptools
+  TAR_NAME: shaptools
+jobs:
+  
+  testPython:
+    runs-on: ubuntu-18.04
+    if: ${{ github.event_name != 'pull_request' }} 
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
+        python-version: [2.7,3.6, 3.7, 3.8, 3.9]
```

Comment:
Add space after `,`. This would be `[2.7, 3.6, 3.7, 3.8, 3.9]`

Reply:
Thanks, I overlooked this